Recipes/Recipes.DataAccess/Interfaces/IGenericRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Recipes.DataAccess.Interfaces;
+
+public interface IGenericRepository<T> where T : class
+{
+    public Task<T> CreateAsync(T entity);
```

Comment:
```suggestion
    public Task<T> InsertAsync(T entity);
```

Nitpick, I think it's just more common, perhaps due to SQL syntax.